core/src/main/java/org/apache/accumulo/core/data/Key.java

```diff
@@ -119,7 +119,10 @@ public Key() {
 
   /**
    * Creates a key with the specified row, empty column family, empty column qualifier, empty column
-   * visibility, timestamp {@link Long#MAX_VALUE}, and delete marker false.
+   * visibility, timestamp {@link Long#MAX_VALUE}, and delete marker false.  This constructor creates
+   * a copy of row. If you don't want to create a copy of row, you should call
```

Comment:
Could be made a bit shorter w/o loss of information.  Something like :

```
To avoid copying, use {@link Key#Key(byte[] row, byte[] cf, byte[] cq, byte[] cv, long ts, boolean deleted, boolean copy)}.
```

Reply:
code updated to use more concise comment